SwissTransferFeatures/NewTransferView/CustomTextField.swift

```diff
@@ -0,0 +1,48 @@
+/*
+ Infomaniak SwissTransfer - iOS App
+ Copyright (C) 2024 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import STResources
+import SwiftUI
+import SwissTransferCoreUI
+
+struct CustomTextField: View {
+    @FocusState private var isMessageFocused: Bool
+    @Binding var value: String
+
+    let placeholder: String
+    var height: CGFloat?
+
+    var body: some View {
+        TextField(placeholder, text: $value, axis: .vertical)
+            .focused($isMessageFocused)
+            .frame(minHeight: height, alignment: .top)
+            .padding(value: .intermediate)
+            .overlay(
+                RoundedRectangle(cornerRadius: 4)
+                    .strokeBorder(Color.ST.textFieldBorder)
+            )
+            .contentShape(.rect)
+            .onTapGesture {
+                isMessageFocused = true
+            }
```

Comment:
Why do we need a tap gesture to focus the textfield?

Reply:
Without this the tap area is really small